dpe/src/commands/sign.rs

```diff
@@ -83,7 +83,7 @@ impl CommandExecution for SignCmd {
     ) -> Result<Response, DpeErrorCode> {
         // Make sure the operation is supported.
         if !dpe.support.is_symmetric() && self.uses_symmetric() {
-            return Err(DpeErrorCode::InvalidArgument);
+            return Err(DpeErrorCode::ArgumentNotSupported);
```

Comment:
All other commands return ArgumentNotSupported. Only Sign comamnd is returning a different error code, so changed it.